src/saturn_engine/worker_manager/services/sync.py

```diff
@@ -28,52 +29,62 @@ def _sync_jobs(
     static_definitions: StaticDefinitions,
     session: AnySyncSession,
 ) -> None:
+    logger = logging.getLogger(__name__)
+
     # Jobs with no interval
     for saturn_job in static_definitions.jobs.values():
```

Comment:
It's just a big index with try/except, it's annoying that github diff doesn't see this...